amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/Lease.java

```diff
@@ -214,6 +222,15 @@ public void pendingCheckpoint(ExtendedSequenceNumber pendingCheckpoint) {
         this.pendingCheckpoint = pendingCheckpoint;
     }
 
+    /**
+     * Sets pending checkpoint state.
+     *
+     * @param pendingCheckpointState can be null
+     */
+    public void pendingCheckpointState(byte[] pendingCheckpointState) {
+        this.pendingCheckpointState = pendingCheckpointState;
```

Comment:
Shouldnt this be setPendingCheckpointState ?

Reply:
I don't have a strong preference, I think `set...` is more readable but the other setters in this class are following the "fluent" naming-scheme, so was keeping this in-line with existing nomenclature.